apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -1699,6 +1699,231 @@ properties:
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_SUBLOCALITY
+    summary: Tell keyboard that content type is, sublocality.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_COUNTRY_NAME
+    summary: Tell keyboard that content type is, country name.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_POSTAL_CODE
+    summary: Tell keyboard that content type is, postal code.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_TELEPHONE_NUMBER
+    summary: Tell keyboard that content type is, telephone number.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_EMAIL_ADDRESSS
+    summary: Tell keyboard that content type is, email address.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_URL
+    summary: Tell keyboard that content type is, url.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_CREDIT_CARD_NUMBER
+    summary: Tell keyboard that content type is, credit card number.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "10.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_USER_NAME
+    summary: Tell keyboard that content type is, user name.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
+    type: String
+    permission: read-only
+    osver: {ios: {min: "11.0"}}
+    since: "6.2.0" 
+
+  - name: CONTENT_TYPE_PASSWORD
+    summary: Tell keyboard that content type is, password.
+    description: |
+        Use with <Titanium.UI.TextField.textContentType> or <Titanium.UI.TextArea.textContentType>.    
```

Comment:
If people use the keyboard-type "email" and the text-fied property "passwordMask", it will enable auto-fill automatically, even when you did not build for iOS 11. That should be noted as an own paragraph there.